extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-service.js

```diff
@@ -122,6 +122,24 @@ describes.realWin(
       }
     );
 
+    it(
+      'should not allow elements to register,' +
+        ' if they do not all or none pass in the global attribute',
+      () => {
```

Comment:
Thanks for calling this out, async / await certainly does look clean.  I went with the promises style this time since the existing tests were all done this way so wanted to be consistent.